pom.xml

```diff
@@ -59,13 +53,8 @@
 			<layout>p2</layout>
 			<url>${jbosstools-base-site}</url>
 		</repository>
-		<repository>
```

Comment:
This one is not to be removed as it is used for retrieve the LSP4MP and Quarkus LSP server as depencies